R/checks.R

```diff
@@ -178,3 +194,28 @@ CheckPredictionVariables <- function(object, newdata)
     }
     return(newdata)
 }
+
+checkPredictionWarningMessage <- function(label, levels, counts, outliers.removed = FALSE)
+{
+    variable.label <- sQuote(label)
+    categories <- ngettext(length(levels), "the category", "categories")
+    context.msg <- paste0("that ", ngettext(length(levels), "was", "were"), " not used in the training data")
+    if (outliers.removed)
+        context.msg <- paste0(context.msg, paste0(" since the automated outlier removal identified those ",
+                                                  "observations as outliers and removed them"))
+    n.counts <- length(counts)
+    if (n.counts == 1)
```

Comment:
How about swapping this block of `if` statements regarding counts with the `context.msg` variable creation so it is more clear to someone looking at the code what the output message actually is?

Reply:
Swapped and added some inline comments in the function.